### DISCUSSION TITLE
Add a better way to filter current playing users

### DISCUSSION THREAD

**Discussion by maotovisk** [ppy/osu#17990] [Ideas] [answered]
Seeing the crescent amount of users that are now logging daily on lazer, it seems that now is a good time to implement a better way to filter players, spectate is getting harder to use due to the amount of users logging in and out at the same time. It would be nice to have a way to filter those users by username and/or friend-list at least.

https://user-images.githubusercontent.com/20617307/165373638-e4a05afc-469b-48b5-b198-1d689784961b.mp4

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/14273
https://github.com/ppy/osu/discussions/17495
https://github.com/ppy/osu/discussions/17740
